java/pom.xml

```diff
@@ -258,7 +289,7 @@
                         <java>
                             <googleJavaFormat>
                                 <version>1.7</version>
-                                <style>AOSP</style>
+                                <style>GOOGLE</style>
```

Comment:
Why change this?

Reply:
I think AOSP is find for JAVA, DO we have any necessary reason to change the format configuration?